.github/scripts/check-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Checking if current tag matches the package version
+current_tag=$(echo $GITHUB_REF | tr -d 'refs/tags/v')
+file1='meilisearch-auth/Cargo.toml'
+file2='meilisearch-http/Cargo.toml'
+file3='meilisearch-lib/Cargo.toml'
+file4='meilisearch-types/Cargo.toml'
+file5='Cargo.lock'
+
+file_tag1=$(grep '^version = ' $file1 | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
+file_tag2=$(grep '^version = ' $file2 | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
+file_tag3=$(grep '^version = ' $file3 | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
+file_tag4=$(grep '^version = ' $file4 | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
+file_tag5=$(grep -A 1 'name = "meilisearch-auth"' $file5 | grep version | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
+
+if [ "$current_tag" != "$file_tag1" ] || [ "$current_tag" != "$file_tag2" ] || [ "$current_tag" != "$file_tag3" ] || [ "$current_tag" != "$file_tag4" ] || [ "$current_tag" != "$file_tag5" ]; then
+  echo "Error: the current tag does not match the version in package file(s)."
+  echo "$file1: found $file_tag1 - expected $current_tag"
+  echo "$file2: found $file_tag2 - expected $current_tag"
+  echo "$file3: found $file_tag3 - expected $current_tag"
+  echo "$file4: found $file_tag4 - expected $current_tag"
+  echo "$file5: found $file_tag5 - expected $current_tag"
+  exit 1
+fi
```

Comment:
```suggestion

file5=$(grep -A 1 'name = "meilisearch-auth"' $file5 | grep version)

for file in $file1 $file2 $file3 $file4 $file5;
do
    file_tag=$(grep '^version = ' $file | cut -d '=' -f 2 | tr -d '"' | tr -d ' ')
    if [ "$current_tag" != "$file_tag" ]; then
      echo "Error: the current tag does not match the version in package file(s)."
      echo "$file: found $file_tag - expected $current_tag"
      exit 1
    fi
done
```

Maybe we could factorize this code a little bit like that what do you think?

Reply:
Yeeeees definitely thank you! Sorry I did this too quickly, your refactor is way better